planning/behavior_path_start_planner_module/src/start_planner_module.cpp

```diff
@@ -936,18 +948,27 @@ std::vector<Pose> StartPlannerModule::searchPullOutStartPoseCandidates(
   return pull_out_start_pose_candidates;
 }
 
-PredictedObjects StartPlannerModule::filterStopObjectsInPullOutLanes(
-  const lanelet::ConstLanelets & pull_out_lanes, const double velocity_threshold) const
+PredictedObjects StartPlannerModule::filterStopObjectsInShoulderLanes(
+  const lanelet::ConstLanelets & shoulder_lanes, const geometry_msgs::msg::Point & current_point,
+  const double velocity_threshold, const double object_check_forward_distance,
+  const double object_check_backward_distance) const
```

Comment:
pull out lanes will be road lanes not just shoulder lanes.

Reply:
so, better to change function and args name

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/5982/commits/365b4f90f2701f4da883b743acd7913630da5349